### PR TITLE
 [team_kai_test_0]: Hello World PR

### PR DESCRIPTION
**Pull request by Rookie-Kai**
### What?

- brief: Hello World Test PR
- author: Rookie-Kai 

### What changes are included in this PR?

建立了对应目录以及本次测试的main.cpp文件

### Are these changes tested?

测试代码为main.cpp，里面包含了Hello World程序

### other information
finished

### CODE REVIEW

**Comment by Light-City** [comment]
[team_Kai_test_0]: -> [team_kai_test_0]: lowercase underscore

**Comment by Light-City** [comment]
don't remove my pr test.

**Comment by Rookie-Kai** [comment]
I have done it, please checkout it.

**Comment by Rookie-Kai** [comment]
> 不要删除我的公关测试。

I have done it, please checkout it.

File: stl/team_aki/kai/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Hello World!" << std::endl;
```
Comment (martinx) [inline stl/team_aki/kai/src/main.cpp:4]: Indent two spaces

File: stl/team_aki/README.md
Comment (martinx) [inline stl/team_aki/README.md:1]: Your team is team_aki and member is kai, not fight_flow & light-city 😄

File: stl/team_aki/kai/src/main.cpp (hunk 1 above)
Comment (Rookie-Kai) [reply stl/team_aki/kai/src/main.cpp:4]: > Indent two spaces

Line 4 is indented in my file

File: stl/team_aki/README.md
Comment (Rookie-Kai) [reply stl/team_aki/README.md:1]: > Your team is team_aki and member is kai, not fight_flow & light-city 😄
Thank you, I have finished the modification, please checkout it

**Comment by martinx** [comment]
LGTM

**Comment by Rookie-Kai** [comment]
> LGTM

Thank You

**Pull request closed without merging**